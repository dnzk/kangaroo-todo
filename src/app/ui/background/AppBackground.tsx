import style from "./background.module.css"

export default function AppBackground() {
  return (
    <div className={style.background}>
      <div className={style.gradient} />
    </div>
  )
}
