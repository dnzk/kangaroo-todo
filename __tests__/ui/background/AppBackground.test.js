import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import AppBackground from "../../../src/app/ui/background/AppBackground"

describe('AppBackground', () => {
  it('renders correctly', () => {
    const { container } = render(<AppBackground />)
    expect(container).toMatchSnapshot()
  })
})
