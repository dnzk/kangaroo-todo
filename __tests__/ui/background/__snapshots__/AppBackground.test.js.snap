// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBackground renders correctly 1`] = `
<div>
  <div
    class="background"
  >
    <div
      class="gradient"
    />
  </div>
</div>
`;
